form/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/global.css" />
+    <script src="/js/index.js" defer></script>
+    <title>Quiz App - Form</title>
+  </head>
+  <body>
+    <header>
+      <h1>Quiz App</h1>
+    </header>
+    <main class="main__form">
+      <h2 class="form-title" id="form-title">Create a new card</h2>
+      <p class="form-decription" id="form-description">
+        Please fill out this form to create a new card.
+      </p>
+      <section class="form-container">
+        <form
+          class="form"
+          data-js="form"
+          aria-labelledby="form-title"
+          aria-describedby="form-description"
+        >
+          <label class="label" for="form-cardQuestion">Your question:</label>
+          <textarea name="form-cardQuestion" rows="3"></textarea>
```

Comment:
I think the textareas can be self closing.
Besides, the `id` is necessary to create a reference to the `label`, not the `name` attribute.

Reply:
It doesn't work without the closing textarea-tag.
You are right about the id. I actually had it there. I must have deleted it at some point for some unclear reason =)